### PR TITLE
Representing Meshery's Broader Capabilities

### LINKED ISSUE #764: Represent Meshery's broader management capabilities
**Current Behavior**
Meshery manages all kinds of cloud native infrastructure. Branding it as a service mesh manager only misrepresents the full-breadth of its capabilities.

**Desired Behavior**
Replace most, but not all, instances of "service mesh" with "cloud native". Examples: "service mesh manager", "service mesh management plane", "multi-mesh manager", "multi-mesh management plane"... these should be replaced with "cloud native manager".

---
<h4>Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources</h4>

- 🎨 Wireframes and designs for Meshery Site in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by eeshaanSA**
Signed-off-by: Eeshaan Sawant <eeshaans1@gmail.com>

**Description**
Changed various instances where Meshery had an outdated definition.

This PR fixes #764 

**Notes for Reviewers**
Open for Changes and suggestions.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: charts/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Meshery Chart
 
-Meshery is the multi-service mesh management plane. This repository include meshery Helm chart.
+Meshery is the Cloud Native management plane. This repository include meshery Helm chart.
```
Comment (leecalcote) [inline charts/README.md:3]: Lowercase, please.

Comment (eeshaanSA) [reply charts/README.md]: will do. Also, shouldn't it be "This repository includes the Meshery Helm chart." ? @leecalcote

**Comment by eeshaanSA** [comment]
If theres anything else that is required to be changed, do let me know.

**Review by harkiratsm — APPROVED** [review]
LGTM 🚀

**Review by leecalcote — APPROVED** [review]
Thank you, gents.

**Pull request merged**